CONTRIBUTING.md

```diff
@@ -113,6 +113,40 @@ the repository maintainers will integrate it, prepare changelogs, and
 ensure it gets released in one of upcoming Weekly releases.
 There is no additional action required from pull request authors at this point.
 
+### Pull-Requests management
+
+The project is using some labels to mark the status and the content of the pull-requests.
+The complete list of labels can be found at https://github.com/jenkinsci/jenkins/labels.
+Out of those labels, here are a definition of labels the maintainers are using to manage the status of the pull-requests:
+
+- `needs-docs` marks a pull-requests lacking documentation in the code or on jenkins.io ;
+such pull-requests won't be merged until the comments are addressed
+- `needs-fix` marks a pull-requests with reviews requesting some code change which are not addressed yet ;
+such pull-requests won't be merged until the code has been fixed and the tests are passing
+- `needs-justification` marks pull-requests on which maintainers are debating the motivation for the proposed changed
+- `needs-more-review` marks pull-requests which are lacking reviews and comments, because the changes are complex or because a debate started among reviewers and more opinions would be beneficial
+- `on-hold` marks pull-requests depending on another event/release, and it cannot be merged right now
+- `propose-for-close` marks pull-requests with no activities or consensus on its content and maintainers do not see a way forward ;
+such pull-requests are usually closed within the next 96hrs after the label is applied
+- `ready-for-merge` marks pull-requests which were reviewed and approved by at least two maintainers ;
```

Comment:
This is not quite accurate according to the maintainer docs and PR checklist. There are exceptions for docs, dependency updates and others.

Reply:
Optional suggestion for your consideration:

"Marks pull requests that have met all the acceptance criteria as defined elsewhere in this document."

Reply:
```suggestion
- `ready-for-merge` marks pull requests that met all the acceptance criteria, as defined elsewhere in this document ;
```